pgbouncer/generate_pgbouncer_config.py

```diff
@@ -39,7 +40,13 @@
 max_db_connections = {BOUNCER_MAX_DB_CONNECTIONS}
 max_user_connections = {BOUNCER_MAX_USER_CONNECTIONS}
 server_idle_timeout = 0
-""")
+"""
+
+config.logg.debug(config_string)
```

Comment:
Should we be logging this?

Reply:
meh probably not lol. it's kinda redundant. will remove